.circleci/config.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - run:
           name: Run golang linter
           command: |
-            golangci-lint run --disable golint,funlen,godox,whitespace,stylecheck --build-tags fuse_cli --max-same-issues 0 --verbose
+            golangci-lint run --disable golint,funlen,godox,whitespace,stylecheck,unparam --build-tags fuse_cli --max-same-issues 0 --verbose
```

Comment:
Why don't we summarize this under `.golangci.yml` and allow contributors to share the same linting rules as CI?

Reply:
Makes sense.

Reply:
and done